dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -821,6 +822,44 @@ portalRouter
   .all(validatePortalFacilityAmendmentsEnabled, validation.mongoIdValidation('facilityId'))
   .put(validatePutPortalFacilityAmendmentPayload, putFacilityAmendmentController.putAmendmentDraft);
 
+/**
+ * @openapi
+ * /deals/:dealId/all-type-amendments:
+ *   get:
+ *     summary: Get all type facility amendments on a given deal
+ *     tags: [Portal - Amendments]
+ *     description: Get all type facility amendments on a given deal
+ *     parameters:
+ *       - in: path
+ *         name: dealId
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: Deal ID to get amendments for
+ *       - in: query
+ *         name: statuses
+ *         schema:
+ *           type: array
+ *           items:
+ *             type: string
+ *           example: ['Draft']
+ *         required: false
+ *         description: The amendment statuses to filter on
+ *     responses:
+ *       200:
+ *         description: OK
+ *         content:
+ *           application/json:
+ *             schema:
+ *                $ref: '#/definitions/PortalAmendment'
+ *       404:
+ *         description: Not found
+ */
+portalRouter
+  .route('/deals/:dealId/all-type-amendments')
```

Comment:
```suggestion
  .route('/deals/:dealId/all-types-amendments')
```
Would this be better? @abhi-markan